### PR TITLE
issue 1 task list completed. #1 - Francis Manalang

### PR DESCRIPTION
**Pull request by francismanalang**
https://user-images.githubusercontent.com/101234537/179639760-8b7ae024-e3c2-4d26-98de-25e739822c51.mov

### CODE REVIEW

**Comment by francismanalang** [comment]
https://user-images.githubusercontent.com/101234537/179640160-c488d36b-c9f0-4012-a018-c35e486f4a55.mov

**Comment by francismanalang** [comment]
https://user-images.githubusercontent.com/101234537/179648802-00b81f96-39fa-424b-9ddb-edda614681d7.mov

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    photoUrl: $entryForm.elements.photoUrl.value,
+    notes: $entryForm.elements.notes.value
+  };
+  formValues.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.unshift(formValues);
+  $image.setAttribute('src', '/images/placeholder-image-square.jpg');
+  $entryForm.elements.title.value = '';
+  $entryForm.elements.photoUrl.value = '';
+  $entryForm.elements.notes.value = '';
```
Comment (codingcodymiller) [inline js/main.js]: I would suggest using the `reset` method on `$entryForm`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+if (previousDataJSON !== null) {
+  JSON.parse(previousDataJSON);
+}
+
+function handleStroage(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('javascript-local-storage', dataJSON);
+}
+
+window.addEventListener('beforeunload', handleStroage);
```
Comment (codingcodymiller) [inline js/main.js]: This should be in `data.js`

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  $entryForm.elements.photoUrl.value = '';
+  $entryForm.elements.notes.value = '';
+}
+
+$entryForm.addEventListener('submit', handleEntryForm);
+
+var previousDataJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousDataJSON !== null) {
+  JSON.parse(previousDataJSON);
```
Comment (codingcodymiller) [inline js/main.js]: You should be assigning this to `data`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
In addition to my other feedback, you are missing the colored outline on each input when they are focused. I suggest using the `outline` property:

https://developer.mozilla.org/en-US/docs/Web/CSS/outline

File: js/main.js (hunk 3 above)
Comment (francismanalang) [reply js/main.js]: https://user-images.githubusercontent.com/101234537/179651474-857245e4-465c-4502-a8e5-ab17fb67b6d6.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**